src/schema.py

```diff
@@ -144,9 +144,10 @@ def resolve_account_info(self, info, session_id=None):
       location = txn['locationName'].rsplit(' ', 1)[0]
       # removes the register numbers at the end of the string by taking the substring up until
       # the last space (right before the number)
+      name = LOCATION_NAMES.get(location, {'name': location})['name']
       new_transaction = {
           'amount': txn['amount'],
-          'name': LOCATION_NAMES.get(location, location),
+          'name': name if name != 'Amit Bhatia Libe Café' else 'Libe Café',
```

Comment:
do we really need additional parsing here? there isn't a way to put this in the previous snippet?

Reply:
bump on this @cswenberg, you can resolve if it's not possible but i don't think we should have two rounds of parsing –– if anything, just put it in an `if else` block and it would probably be cleaner